Makefile

```diff
@@ -69,21 +69,21 @@ CLOUDINIT_GENERATED := $(CLOUDINIT_PKG_DIR)/zz_generated.bindata.go
 CLOUDINIT_SCRIPT := $(CLOUDINIT_PKG_DIR)/kubeadm-bootstrap-script.sh
 
 # Define Docker related variables. Releases should modify and double check these vars.
-REGISTRY ?= gcr.io/$(shell gcloud config get-value project)
+REGISTRY ?= quay.io
```

Comment:
dockerhub.io ?

Reply:
Later - I don't know how fast the syncing is

Reply:
Or do you know how we can push to dockerhub directly?